packages/harmony/src/components/comments/IconText/IconText.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex } from 'components/layout'
+import { Text } from 'components/text'
+
+import { IconTextProps } from './types'
+
+export const IconText = ({
+  icons = [],
```

Comment:
nit: I would personally prefer if we also offered both `icon` and `icons` as props (and just mutually exclude them in types) More intuitive and I think singular icon will be the norm more than plural (although not entirely sure how often this component gets used tbh)

Reply:
yea this is just an internal comments component won't be used much outside of building the other components, so fine to leave for now.

Reply:
yeah on second thought not worth; I realized you even separated this in a "comments" section in harmony. Makes sense 👌